packages/engine.io-client/src/stellarSocket.js

```diff
@@ -114,17 +116,17 @@ function stellarSocketFactory(eio) {
         }
       });
     },
-    _doConnect(url, { userId, token, secure, tokenType, eioConfig = { upgrade: true, rememberUpgrade: true } }) {
+    _doConnect(url, { userId, token, secure, tokenType, params, eioConfig = { upgrade: true, rememberUpgrade: true } }) {
       log.info(`@StellarEngineIO._doConnect: ${userId}, ${token}`);
       return new Promise((resolve, reject) => {
         this.state = 'connecting';
-
+        const urlParams = _.assign({}, params,
+            { 'x-auth-user': userId, 'x-auth-token': token, 'x-auth-token-type': tokenType });
```

Comment:
can you add a test to `bridge/tests/clientserver.test.js` for url parameter sending